fe/src/lib/components/svizzle/Grid2Columns.svelte

```diff
@@ -1,13 +1,13 @@
 <script>
 	import {makeStyleVars} from '@svizzle/dom';
 
-	export let gap;
-	export let percents = [50, 50];
+	export let gap = 0;
```

Comment:
I'd make sure gap is never undefined (`$: gap = gap ?? 0`) in all of the `Grid*` components, in case we update the prop from something to nothing